tests/Functional/Schema/Snowflake/SnowflakeSchemaQueryBuilderTest.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Keboola\TableBackendUtils\Functional\Schema\Snowflake;
+
+use Doctrine\DBAL\Exception;
+use Keboola\TableBackendUtils\DataHelper;
+use Keboola\TableBackendUtils\Escaping\Snowflake\SnowflakeQuote;
+use Keboola\TableBackendUtils\Schema\Snowflake\SnowflakeSchemaQueryBuilder;
+use Tests\Keboola\TableBackendUtils\Functional\Connection\Snowflake\SnowflakeBaseCase;
+
+/**
+ * @covers SnowflakeSchemaQueryBuilder
+ */
+class SnowflakeSchemaQueryBuilderTest extends SnowflakeBaseCase
+{
+    public const TEST_SCHEMA = self::TESTS_PREFIX . 'qb_schema_schema';
+    public const TEST_SCHEMA_2 = self::TESTS_PREFIX . 'qb_schema_schema2';
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $this->cleanSchema(self::TEST_SCHEMA);
+        $this->cleanSchema(self::TEST_SCHEMA_2);
+    }
+
+    public function testGetCreateSchemaCommand(): void
+    {
+        $qb = new SnowflakeSchemaQueryBuilder();
+        $schemasNumberBefore = count($this->getSchemaFromDatabase());
+
+        $this->connection->executeQuery($qb->getCreateSchemaCommand(self::TEST_SCHEMA));
+
+        $schemas = $this->getSchemaFromDatabase();
+        self::assertCount($schemasNumberBefore + 1, $schemas);
+        self::assertContains(self::TEST_SCHEMA, $schemas);
+    }
+
+    /**
+     * @return string[]
+     */
+    private function getSchemaFromDatabase(): array
+    {
+        $schemas = $this->connection->fetchAllAssociative(
+            'SHOW SCHEMAS'
+        );
+
+        return DataHelper::extractByKey($schemas, 'name');
+    }
+
+    public function testGetDropSchemaCommandWithCascade(): void
+    {
+        $schemasNumberBefore = count($this->getSchemaFromDatabase());
+        $this->cleanSchema(self::TEST_SCHEMA_2);
```

Comment:
Preco tu volas este raz `$this->cleanSchema(self::TEST_SCHEMA_2);` ked to je aj v `setUp()`  co sa vola pred kazdym testom